FWCore/PyDevParameterSet/interface/PyBind11ProcessDesc.h

```diff
@@ -13,6 +13,8 @@ namespace edm {
   class ProcessDesc;
 }
 
+namespace cmspython3 {
```

Comment:
this namespace is practically creating a parallel world for the python3 version of the framework interpreter. I assume this is for the transition period, asymptotically this will converge back to edm, am I correct?